Directory.Build.rsp

```diff
@@ -0,0 +1,4 @@
+/ConsoleLoggerParameters:Verbosity=Minimal;Summary;ForceNoAlign
```

Comment:
`ForceNoAlign` exactly what is it solving? 
I looked into [doc](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-command-line-reference?view=vs-2022) but doesn't have any example. It says `Don't align the text to the size of the console buffer.` If buffer filled then exactly what happens to user experience?

Reply:
By default, MSBuild places line breaks in console messages to match the current width of the window.  This means if you run `MSBuild.exe` and then resize the window, the message remain at the same width.  It also means that if you copy/paste a message from that console, it will copy the line breaks.  ForceNoAlign tells MSBuild to just log messages without line breaks and let the window wrap or not wrap the output.  It also makes it so you can copy/paste the output without line breaks in it.